sui/deploy-contract.js

```diff
@@ -127,7 +127,7 @@ async function postDeployExample(published, keypair, client, config, chain, opti
     const [gmpSingletonObjectId] = getObjectIdsByObjectTypes(published.publishTxn, [`${published.packageId}::gmp::Singleton`]);
 
     // ITS Example Params
-    const itsObjectId = config.sui.contracts.ITS?.objects?.ITS;
+    const itsObjectId = config.chains.sui.contracts.ITS?.objects?.ITS;
```

Comment:
use `getChainConfig` instead of hardcoding the name. See what we do for stellar
https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/stellar/deploy-contract.js#L101

Reply:
chain config should be retrieved in the mainProcessor and provided to this function so you only need to use the helper once